verilog/systemverilog/SystemVerilogParser.g4

```diff
@@ -233,7 +235,8 @@ elaboration_system_task
 	;
 
 finish_number
-	: FINISH_NUMBER
+	: //FINISH_NUMBER
```

Comment:
Could commented code be removed? Here and below.

Reply:
I'd like to keep those comments to show the changes I've made to the original syntax. New versions of the rules are indented one extra level to the right.